### DISCUSSION TITLE
Turning Theme

### DISCUSSION THREAD

**Discussion by thiagotw** [mathoudebine/turing-smart-screen-python#408] [Themes]
I translate and made some variations from the official Turning.data theme

Using the UsbMonitor.exe, go to Theme Editor option, load the original Turning theme, load the UI, set a name and save.
Tested on 3.5

Red Version

![theme tuning red](https://github.com/mathoudebine/turing-smart-screen-python/assets/27924949/c49fb8e7-331a-4246-bc8f-5cb767ecff0a)

Green Version

![theme tuning green](https://github.com/mathoudebine/turing-smart-screen-python/assets/27924949/5774b1ac-8a71-46a6-ac05-a79c9a5671ad)

**Comment by matthewric** [comment]
Where do I get your theme fome mine won't edit properly

**Comment by nirrrrrrrrrrrrrrrrrrr** [comment]
i want this

**Comment by c0ldJS** [comment]
> "Using the UsbMonitor.exe,..."

You're using the stock driver for your panel. Folks here are not.
